### DISCUSSION TITLE
Competitive bumpkin rank system boost

### DISCUSSION THREAD

**Discussion by alivitedg** [sunflower-land/sunflower-land#1999] [Ideas]
**### Proposal bumpkin boost by rarity**

There are 6 rarity:

  1. Legendary
  2. Epic
  3. Super Rare
  4. Rare
  5. Uncommon
  6. Common

We give a boost to the bumpkin and it will be the boost different between rarity and other rarity example: we give the 1st rank x3 crop and fruit yield  and the 100st x2 crop and fruit yield 
And the 501st x1. 2 crop yield without boost in fruit.
We give the legendary and the epic boost in crop and fruit and the other category the boost just in the crop so the first 200 have boosted in crops and fruit and the rank larger than 200 we give he boost just in crop without a boost in fruit.

It's competitive system and this system will give high value to bumpkins and Bumpkin Wearables.

Here is how the classification will be by rank :
		1 -> 100    : legendary   -> boost ( x2 to x3 in crop and fruit yield)
		101 -> 200  : epic     	  -> boost ( x1.8 to x2 in crop and fruit yield)
		201 -> 300  : super rare  -> boost ( x1.6 to x1.8 in crop yield)
		301 -> 500  : rare	  -> boost ( x1.2 to x1.6 in crop yield)
		501 -> 700  : uncommon    -> boost ( x1.2 in crop yield)
		701-> 1000  : common	  -> boost ( x1.1 in crop yield)
		rank > 1000 : common	  -> without boost

![rank1](https://user-images.githubusercontent.com/106750574/211206282-cf68a662-89c1-462d-b581-f2bdb5906dbb.png)

Here is the distribution of the boost by rank:

**Legendary (from rank 1 to rank 100)**
1 -> 100 	:	x3 -> x2
	1 : x3			crop and fruit yield
	2 : x2.99		crop and fruit yield
	3 : x2.98		crop and fruit yield
	4 : x2.97		crop and fruit yield
	5 : x2.96		crop and fruit yield
	6 : x2.95		crop and fruit yield
	7 : x2.94
	8 : x2.93
	9 : x2.92
	10 : x2.91
	11 : x2.9
	12 : x2.89
	13 : x2.88
	14 : x2.87
	15 : x2.86
	16 : x2.85
	17 : x2.84
	18 : x2.83
	19 : x2.82
	20 : x2.81
	21 : x2.8
	.
	.
	.
       98 : x2.03       crop and fruit yield
       99 : x2.02       crop and fruit yield
       100 : x2.01     crop and fruit yield

**Epic (from rank  101 to rank 200)**
	101 -> 110      :       x1.99 crop and fruit yield
	111 -> 120      :       x1.97 crop and fruit yield
	121 -> 130      :       x1.95 crop and fruit yield
	131 -> 140      :       x1.93 crop and fruit yield
	141 -> 150      :       x1.91 crop and fruit yield
	151 -> 160      :       x1.89 crop and fruit yield
	161 -> 170      :       x1.87 crop and fruit yield
	171 -> 180      :       x1.85 crop and fruit yield
	181 -> 190      :       x1.83 crop and fruit yield
	191 -> 200      :       x1.81 crop and fruit yield

	
**Super Rare (from rank  201 to rank 300)**
	201 -> 210	:	x1.79 crop yield
	211 -> 220	:	x1.77 crop yield
	221 -> 230	:	x1.75 crop yield
	231 -> 240	:	x1.73 crop yield	
	241 -> 250	:	x1.71 crop yield
	251 -> 260	:	x1.69 crop yield
	261 -> 270	:	x1.67 crop yield
	271 -> 280	:	x1.65 crop yield
	281 -> 290	:	x1.63 crop yield
	291 -> 300	:	x1.61 crop yield

**Rare (from rank  301 to rank 500)**
	301 -> 310	:	x1.59 crop yield
	311 -> 320	:	x1.57 crop yield
	321 -> 330	:	x1.55 crop yield
	331 -> 340	:	x1.53 crop yield
	341 -> 350	:	x1.51 crop yield
	351 -> 360	:	x1.49 crop yield
	361 -> 370	:	x1.47 crop yield
	371 -> 380	:	x1.45 crop yield
	381 -> 390	:	x1.43 crop yield
	391 -> 400	:	x1.41 crop yield
	401 -> 310	:	x1.39 crop yield
	411 -> 320	:	x1.37 crop yield
	421 -> 330	:	x1.35 crop yield
	431 -> 340	:	x1.33 crop yield
	441 -> 350	:	x1.31 crop yield
	451 -> 360	:	x1.29 crop yield
	461 -> 370	:	x1.27 crop yield
	471 -> 380	:	x1.25 crop yield
	481 -> 390	:	x1.23 crop yield
	491 -> 500	:	x1.21 crop yield

		
**Uncommon(from rank 501 to rank 700)**
	501 -> 700	:	x1.2 crop yield

**Common (from rank 701 to rank 1000)**
	701 -> 1000		:	x1.1 crop yield

**Common (rank bigger than 1000)**
	rank > 1000		:	without boost
	
**Sorry for the language mistakes.**

**Comment by cryptosmikey** [comment]
This is a really awesome idea. As the game progression moves on and new player flood in down the road this will keep the interest and allow even more room for things to flow. nicely done!
